src/main/java/seedu/address/logic/commands/TagCommand.java

```diff
@@ -97,9 +131,34 @@ public CommandResult addUserTags(Model model) throws CommandException {
         model.getObservablePersonList().forEach(person ->
                 person.setCommonModules(editedUser.getImmutableModuleTags()));
 
+        for (ModuleTag tag : moduleTags) {
+            String day = tag.getDay();
+            String start = tag.getStartTime();
+            String end = tag.getEndTime();
+            if (day == null || start == null || end == null) {
+                continue;
+            }
+
+            Module mod = new Module(tag.toString());
+            int startHour = Integer.parseInt(start.substring(0, 2));
+            int startMin = Integer.parseInt(start.substring(2));
+            int endHour = Integer.parseInt(end.substring(0, 2));
+            int endMin = Integer.parseInt(end.substring(2));
+            LocalTime startTime = new LocalTime(startHour, startMin);
+            LocalTime endTime = new LocalTime(endHour, endMin);
```

Comment:
Please note, we plan the minutes to be 00 for ALL classes. That is due to the design decision made for `Timetable`

Reply:
What about inputs? Is it wise to just let users to give the 2 digit hour as an input? I am fine with either way.

Reply:
If the start time is 0930, we could count it as 0900. If the end time is 1030, we could count it as 1100.

Reply:
That would be the ideal handling of non-hour blocks.